pulp_2to3_migration/app/plugin/content.py

```diff
@@ -246,16 +249,21 @@ def get_remote_by_importer_id(importer_id):
                 select_extra.append('pulp2content__pulp2_repo')
 
             pulp_2to3_detail_qs = pulp_2to3_detail_qs.select_related(*select_extra)
-            for pulp_2to3_detail_content in pulp_2to3_detail_qs.iterator(chunk_size=800):
+            async for pulp_2to3_detail_content in sync_to_async_iterable(
+                    pulp_2to3_detail_qs.iterator(chunk_size=800)
+            ):
                 dc = None
                 pulp2content = pulp_2to3_detail_content.pulp2content
+                await sync_to_async(pulp2content.refresh_from_db)()  # force model instance to be
```

Comment:
What's going on here?

Reply:
pulp2content is used in many places later, this is to force to load fully from the db without a need to do db requests later when different fields are accessed.

Reply:
I think whether get() or refresh_from_db() is used there is not much of the difference. refresh_from_db() calls anyway into get() its just does not load deferred fields and clears cached relations https://github.com/django/django/blob/main/django/db/models/base.py#L652